### PR TITLE
Issue 47: Update CLI -r flag description

### PR DESCRIPTION
**Pull request by jonlouie**
## Related issue

Closes: #47 

## Description
* Update CLI `-r` flag description to match its functionality (currently labeled as a file when it is used as a directory)
* Update any downstream variable names affected by this

## Supplemental testing
Describe any testing done in addition to existing unit tests

## Additional context
Please provide any additional information related to this PR

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: src/CTA.Rules.PortCore/PortCoreRulesCli.cs [hunk 1]
```diff
@@ -12,8 +12,8 @@ public class PortCoreOptions
         [Option('s', "solution-path", Required = false, HelpText = "Solution file path.")]
         public string SolutionPath { get; set; }
 
-        [Option('r', "rules-input-infile", Required = false, HelpText = "Rules json input file")]
-        public string RulesInputFile { get; set; }
+        [Option('r', "rules-input-infile-directory", Required = false, HelpText = "Directory containing rules json input file(s)")]
+        public string RulesInputFilesDirectory { get; set; }
```
Comment (marknfawaz) [inline src/CTA.Rules.PortCore/PortCoreRulesCli.cs:16]: nit: We can shorten the names to rules-dir and RulesDir to stay consistent with the other variables.

File: src/CTA.Rules.Update/RulesCli.cs [hunk 2]
```diff
@@ -13,8 +13,8 @@ public class Options
         [Option('s', "solution-path", Required = false, HelpText = "Solution file path.")]
         public string SolutionPath { get; set; }
 
-        [Option('r', "rules-input-infile", Required = false, HelpText = "Rules json input file")]
-        public string RulesInputFile { get; set; }
+        [Option('r', "rules-input-infile-directory", Required = false, HelpText = "Directory containing rules json input file(s)")]
+        public string RulesInputFilesDirectory { get; set; }
```
Comment (marknfawaz) [inline src/CTA.Rules.Update/RulesCli.cs:17]: nit: Same as comment for PortCore project

**Review by marknfawaz — CHANGES_REQUESTED** [review]

File: src/CTA.Rules.PortCore/PortCoreRulesCli.cs (hunk 1 above)
Comment (jonlouie) [reply src/CTA.Rules.PortCore/PortCoreRulesCli.cs:16]: Good suggestion, will do.

File: src/CTA.Rules.Update/RulesCli.cs (hunk 2 above)
Comment (jonlouie) [reply src/CTA.Rules.Update/RulesCli.cs:17]: Will do

**Review by marknfawaz — APPROVED** [review]

**Pull request merged**